pkg/util/ipvs/ipvs_linux.go

```diff
@@ -214,10 +214,14 @@ func toVirtualServer(svc *ipvs.Service) (*VirtualServer, error) {
 		Port:      svc.Port,
 		Scheduler: svc.SchedName,
 		Protocol:  protocolNumbeToString(ProtoType(svc.Protocol)),
-		Flags:     ServiceFlags(svc.Flags),
 		Timeout:   svc.Timeout,
 	}
 
+	if svc.Flags < uint32(FlagHashed) {
```

Comment:
If flags are treated as a bitmap, we should use bit tests for them,

`if svc.Flags & FlagHashed == 0`, and so on.

Reply:
Thanks. Done. PTAL.